app/components/ToggleDisplay.js

```diff
@@ -80,28 +79,17 @@ export default class ToggleDisplay extends Component {
 
   runHelpers = () => {
     if (this.state.helpers) {
```

Comment:
Okay with this bit I must admit I was just super confused. I might've completely misunderstood the code but it just seemed to make no sense to me. Again a `.map` is used without using the return value, we're again reassigning an object pointer to mutate it, and then in the old lines 99-104 we're only mutating the last helper from the array (as obj keeps getting overwritten every time a hide helper is shown) before we only send this last helper through to the DOM. It seems very confusing and to be honest completely nonsensical, but it might be I'm missing something.

Also, just to make sure I understand what helpers are, they are elements of html that are connected to the element we actually want to hide that are so connected they should also be hidden, yeah?

If anyone could help shed some light on this that would be great, whether I'm completely off and just misunderstood everything, or it's just a weird oversight, because if I'm right this code was not doing what it was supposed to except for the case that no group had more than 1 helper so it didn't break.